src/comprehensions.py

```diff
@@ -11,22 +11,27 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
```

Comment:
list comprehensions function differently than regular iterations, all of the logic is included inside (compressed) a list, hence the name.

This one for example would be, [y.append(x) for x in range(1, 6)]

Reply:
Thanks! It's all fixed now.